src/com/ableton/VirtualEnv.groovy

```diff
@@ -0,0 +1,31 @@
+package com.ableton
+
+
+class VirtualEnv implements Serializable {
+  def script
+  def destDir
+
+  VirtualEnv(script) {
+    this.script = script
+    def pathSeparator = script.isUnix() ? "/" : "\\"
+    this.destDir = script.pwd(tmp:true) + pathSeparator + script.env.BUILD_NUMBER
```

Comment:
We should maybe add `python` to the `destDir`, so a given job can "compare" two Python versions. However, if we consider that a given job can create only one venv, then we should simply use a singleton.

Reply:
I really like this idea, if for no other reason than we'd see the python version in the logs for any `sh` calls.